lib/github_service/commands/cross_repo_test.rb

```diff
@@ -177,7 +177,9 @@ def normalize_repo_list(repo_list)
 
       def normalize_repo_name(repo)
         repo = repo.strip
-        repo.include?("/") ? repo : "#{issue.organization_name}/#{repo}"
+        repo = "#{issue.repo_name}#{repo}" if repo.start_with?("#")
```

Comment:
If we're going to handle bare PR numbers should we also do the same for `start_with?("@")` for branches/shas?

Reply:
No, because you can't specify a bare branch - that would be an `@-mention`

Reply:
https://github.com/ManageIQ/manageiq/pull/21060#issuecomment-780668127
https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/323/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R21

Reply:
Ok, I can add it in...it's really weird, but at least the normalization would be "normal"

Reply:
Yeah its really just another case of "Repository doesn't exist" so maybe not even worth it